pyproject.toml

```diff
@@ -27,6 +27,11 @@ cryptography = ">=3.4.8"
 tqdm = "^4.32.1"
 pydantic = ">=1.10.14"
 orjson = ">=2"
+pycocotools = {version = "^2.0.7", optional = true}
```

Comment:
Have you checked that a default install doesn't bring these in? I would have expected these to sit in their own group rather than be in dependencies marked as optional

Reply:
Tagging packages as optional looks like the standard practice according to the [Poetry docs](https://python-poetry.org/docs/pyproject/#extras) while seems like there are other ways to achieve the same result (within the same documentation).

And yes, they don't get installed because they are tagged as optional and the default dependency installation doesn't add extra packages. This behaviour can be tested via 'pip install encord' and 'pip install encord[coco]' on fresh virtual environments. I have tested it but a second try is never misplaced.